### ISSUE TITLE
Issues with Object.prototype.patch

### ISSUE THREAD

**Issue by lee-houghton** [Olegas/node-event-loop-monitor#2]
I have a project while uses both `event-loop-monitor` and the popular `request` library (https://github.com/request/request). I found that using both modules causes problems:

```
$ node
> require("event-loop-monitor");require("request")
TypeError: Cannot assign to read only property 'patch' of function request(uri, options, callback) {
  if (typeof uri === 'undefined') {
    throw new Error('undefined is not a valid uri or options object.')
  }

  var params = initParams(uri, options, callback)
  options = params.options
  options.callback = params.callback
  options.uri = params.uri

  return new request.Request(options)
}
    at Object.<anonymous> (E:\SVN\RestAPI\node_modules\request\index.js:89:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at repl:1:31
    at REPLServer.self.eval (repl.js:110:21)
    at repl.js:249:20
>
```

I think the problem is this particular line of code:
https://github.com/Olegas/node-event-loop-monitor/blob/master/lib/CartesianTree.js#L4

It seems to make Object.prototype.patch read-only, so `request`'s perfectly valid attempt to set `patch` on its own prototype fails.

I may submit a pull request in the future, but I don't currently have a solution, I just wanted to make you aware of the possible compatibility issues.

Requiring the modules in the opposite order is a workaround, but is not desirable, and in my case is a pain since these modules are loaded dynamically.

**Comment by Olegas** [comment]
Thank you for bug report. 
Fixed in 599f819a07c7b78360f86f6d1c9d1d84280d3f65
Please update from NPM to 0.0.4 and check.

**Comment by lee-houghton** [comment]
This has fixed the issue for me. Thanks for the quick fix!
